docs/standard/linq/add-elements-attributes-nodes-xml-tree.md

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
the redirect for this page doesn't seem to be on the changes for this PR

Reply:
The redirection file entries that are missing were in earlier PRs - 37). and 38). I resubmitted some of the files of those PRs, but not the redirection entries, in this PR to make some fixes. I think you can see the redirection entries now that I have resolved merge conflicts.